include/swift/ABI/Executor.h

```diff
@@ -64,6 +64,12 @@ class ExecutorRef {
     return ExecutorRef(reinterpret_cast<uintptr_t>(actor) | IsDefaultActor);
   }
 
+  static ExecutorRef fromOpaque(void *ptr) {
+    return ExecutorRef((uintptr_t)ptr);
+  }
```

Comment:
Do you think we should put an assert here to make sure the mask bits are zero for this pointer?